packages/abstractions/src/serialization/serializationWriter.ts

```diff
@@ -19,7 +19,7 @@ export interface SerializationWriter {
    * @param key the key to write the value with.
    * @param value the value to write to the stream.
    */
-  writeStringValue(key?: string | undefined, value?: string | undefined): void;
```

Comment:
please revert this change, we want the serialization writer interfaces to be consistent across languages

Reply:
I agree